src/nile/core/declare.py

```diff
@@ -26,3 +49,8 @@ def alias_exists(alias, network):
     """Return whether an alias exists or not."""
     existing_alias = next(deployments.load_class(alias, network), None)
     return existing_alias is not None
+
+
+def get_hash(contract_class):
```

Comment:
Is contract class a concept that users are expected to use / be familiar with?  Would it be simpler if this takes a `contract_name` (and perhaps an `overriding_path`) to be consistent with `declare`, and have this function load the artifact instead of requiring the user to do so?

Reply:
The `contract_name` approach seems like a good idea^.

Also, it's a little unclear to me how this function should be used. I assume the intent as-is requires a direct import. If so, would `utils` be a better spot? Or should we add this to the NRE?

Reply:
i'll move it to utils then. usage is described in https://github.com/OpenZeppelin/nile/issues/205. I don't think utils in general would make good commands, although it depends on usage/demand.